packages/@react-spectrum/radio/docs/Radio.mdx

```diff
@@ -47,47 +48,69 @@ category: Forms
-have a single radio by itself, wrap it in a RadioGroup.
+**Note:** A Radio cannot be used outside of a RadioGroup.
 
 ## Value
 
 RadioGroup only allows single selection. An initial, uncontrolled value can be provided to the RadioGroup using the `defaultValue` prop.
 Alternatively, a controlled value can be provided using the `value` prop.
 
-### Accessibility
-
-For accessibility, all Radio elements within a RadioGroup automatically receive an `aria-checked` attribute
-reflecting their individual selection status.
-
 ```tsx example
-<RadioGroup label="Uncontrolled" defaultValue="dragon">
- <Radio value="wizard">Wizard</Radio>
- <Radio value="dragon">Dragon</Radio>
-</RadioGroup>
+function Example() {
+  let [selected, setSelected] = React.useState('yes');
 
-<RadioGroup label="Controlled" value="dragon">
- <Radio value="wizard">Wizard</Radio>
- <Radio value="dragon">Dragon</Radio>
-</RadioGroup>
+  return (
+    <Flex>
+      <RadioGroup label="Are you a wizard? (uncontrolled)" defaultValue="yes" marginEnd="size-300">
+        <Radio value="yes">Yes</Radio>
+        <Radio value="no">No</Radio>
+      </RadioGroup>
+
+      <RadioGroup label="Are you a wizard? (controlled)" value={selected} onChange={setSelected}>
+        <Radio value="yes">Yes</Radio>
+        <Radio value="no">No</Radio>
+      </RadioGroup>
+    </Flex>
+  );
+}
 ```
 
 ## Labeling
-
-When a visible label isn't being used, an `aria-label` prop is needed to explicitly set the label for the RadioGroup.
-Radio elements should always have a visible label.
+A visual label should be provided for the TextField using the `label` prop. If the TextField is required, the `isRequired` and
+`necessityIndicator` props can be used to show a required state.
 
 ```tsx example
-<RadioGroup aria-label="Favorite Avatar">
- <Radio value="wizard">Wizard</Radio>
- <Radio value="dragon">Dragon</Radio>
-</RadioGroup>
+<Flex>
+  <RadioGroup label="Favorite avatar" marginEnd="size-300">
+    <Radio value="wizard">Wizard</Radio>
+    <Radio value="dragon">Dragon</Radio>
+  </RadioGroup>
+
+  <RadioGroup label="Favorite avatar" isRequired necessityIndicator="icon" marginEnd="size-300">
+    <Radio value="wizard">Wizard</Radio>
+    <Radio value="dragon">Dragon</Radio>
+  </RadioGroup>
+
+  <RadioGroup label="Favorite avatar" isRequired necessityIndicator="label" marginEnd="size-300">
+    <Radio value="wizard">Wizard</Radio>
+    <Radio value="dragon">Dragon</Radio>
+  </RadioGroup>
+
+  <RadioGroup label="Favorite avatar" necessityIndicator="label">
+   <Radio value="wizard">Wizard</Radio>
+   <Radio value="dragon">Dragon</Radio>
+  </RadioGroup>
+</Flex>
 ```
+### Accessibility
+
+If a visible label isn't specified, an `aria-label` must be provided to the RadioGroup for accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using the id of the labeling element instead.
+Radio elements should always have a visible label.
```

Comment:
Is this supposed to be a separate paragraph? If so, then it needs another line before it.